<!doctype html>



  


<html class="theme-next mist use-motion" lang="zh-Hans">
<head>
  <meta charset="UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=edge" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1"/>



<meta http-equiv="Cache-Control" content="no-transform" />
<meta http-equiv="Cache-Control" content="no-siteapp" />















  
  
  <link href="/lib/fancybox/source/jquery.fancybox.css?v=2.1.5" rel="stylesheet" type="text/css" />




  
  
  
  

  
    
    
  

  

  

  

  

  
    
    
    <link href="//fonts.googleapis.com/css?family=Lato:300,300italic,400,400italic,700,700italic&subset=latin,latin-ext" rel="stylesheet" type="text/css">
  






<link href="/lib/font-awesome/css/font-awesome.min.css?v=4.6.2" rel="stylesheet" type="text/css" />

<link href="/css/main.css?v=5.1.0" rel="stylesheet" type="text/css" />


  <meta name="keywords" content="文献导读," />





  <link rel="alternate" href="/atom.xml" title="Things worth spreading!" type="application/atom+xml" />




  <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico?v=5.1.0" />






<meta name="description" content="Two extended haplotype blocks are associated with adaptation to high altitude habitats in East African honey bees &amp;gt;&amp;gt;&amp;gt;PLoS Genet 13(5): e1006792.作者通过对东非意蜂高海拔和低海拔群体进行全基因组测序，发现高海拔和低海拔群体基因组之间存在两个">
<meta name="keywords" content="文献导读">
<meta property="og:type" content="article">
<meta property="og:title" content="文献速递 (2017年5月)">
<meta property="og:url" content="http://www.colyfu.com/paper-express/paper-express-201705.html">
<meta property="og:site_name" content="Things worth spreading!">
<meta property="og:description" content="Two extended haplotype blocks are associated with adaptation to high altitude habitats in East African honey bees &amp;gt;&amp;gt;&amp;gt;PLoS Genet 13(5): e1006792.作者通过对东非意蜂高海拔和低海拔群体进行全基因组测序，发现高海拔和低海拔群体基因组之间存在两个">
<meta property="og:updated_time" content="2017-06-03T10:38:53.000Z">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="文献速递 (2017年5月)">
<meta name="twitter:description" content="Two extended haplotype blocks are associated with adaptation to high altitude habitats in East African honey bees &amp;gt;&amp;gt;&amp;gt;PLoS Genet 13(5): e1006792.作者通过对东非意蜂高海拔和低海拔群体进行全基因组测序，发现高海拔和低海拔群体基因组之间存在两个">



<script type="text/javascript" id="hexo.configurations">
  var NexT = window.NexT || {};
  var CONFIG = {
    root: '/',
    scheme: 'Mist',
    sidebar: {"position":"left","display":"post","offset":12,"offset_float":0,"b2t":false,"scrollpercent":false},
    fancybox: true,
    motion: true,
    duoshuo: {
      userId: '0',
      author: '博主'
    },
    algolia: {
      applicationID: '',
      apiKey: '',
      indexName: '',
      hits: {"per_page":10},
      labels: {"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}
    }
  };
</script>



  <link rel="canonical" href="http://www.colyfu.com/paper-express/paper-express-201705.html"/>





  <title> 文献速递 (2017年5月) | Things worth spreading! </title>
</head>

<body itemscope itemtype="http://schema.org/WebPage" lang="zh-Hans">

  





  <script type="text/javascript">
    var _hmt = _hmt || [];
    (function() {
      var hm = document.createElement("script");
      hm.src = "https://hm.baidu.com/hm.js?da55ad3ecaab6360dc40a9dc42aaa258";
      var s = document.getElementsByTagName("script")[0];
      s.parentNode.insertBefore(hm, s);
    })();
  </script>










  
  
    
  

  <div class="container sidebar-position-left page-post-detail ">
    <div class="headband"></div>

    <header id="header" class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-wrapper">
  <div class="site-meta ">
    

    <div class="custom-logo-site-title">
      <a href="/"  class="brand" rel="start">
        <span class="logo-line-before"><i></i></span>
        <span class="site-title">Things worth spreading!</span>
        <span class="logo-line-after"><i></i></span>
      </a>
    </div>
      
        <h1 class="site-subtitle" itemprop="description">Knowledge is a good thing, and good things are worth spreading!</h1>
      
  </div>

  <div class="site-nav-toggle">
    <button>
      <span class="btn-bar"></span>
      <span class="btn-bar"></span>
      <span class="btn-bar"></span>
    </button>
  </div>
</div>

<nav class="site-nav">
  

  
    <ul id="menu" class="menu">
      
        
        <li class="menu-item menu-item-home">
          <a href="/" rel="section">
            
              <i class="menu-item-icon fa fa-fw fa-home"></i> <br />
            
            首页
          </a>
        </li>
      
        
        <li class="menu-item menu-item-categories">
          <a href="/categories" rel="section">
            
              <i class="menu-item-icon fa fa-fw fa-th"></i> <br />
            
            分类
          </a>
        </li>
      
        
        <li class="menu-item menu-item-about">
          <a href="/about" rel="section">
            
              <i class="menu-item-icon fa fa-fw fa-user"></i> <br />
            
            关于
          </a>
        </li>
      
        
        <li class="menu-item menu-item-archives">
          <a href="/archives" rel="section">
            
              <i class="menu-item-icon fa fa-fw fa-archive"></i> <br />
            
            归档
          </a>
        </li>
      
        
        <li class="menu-item menu-item-tags">
          <a href="/tags" rel="section">
            
              <i class="menu-item-icon fa fa-fw fa-tags"></i> <br />
            
            标签
          </a>
        </li>
      

      
        <li class="menu-item menu-item-search">
          
            <a href="javascript:;" class="popup-trigger">
          
            
              <i class="menu-item-icon fa fa-search fa-fw"></i> <br />
            
            搜索
          </a>
        </li>
      
    </ul>
  

  
    <div class="site-search">
      
  <div class="popup search-popup local-search-popup">
  <div class="local-search-header clearfix">
    <span class="search-icon">
      <i class="fa fa-search"></i>
    </span>
    <span class="popup-btn-close">
      <i class="fa fa-times-circle"></i>
    </span>
    <div class="local-search-input-wrapper">
      <input autocapitalize="off" autocomplete="off" autocorrect="off"
             placeholder="搜索..." spellcheck="false"
             type="text" id="local-search-input">
    </div>
  </div>
  <div id="local-search-result"></div>
</div>



    </div>
  
</nav>



 </div>
    </header>

    <main id="main" class="main">
      <div class="main-inner">
        <div class="content-wrap">
          <div id="content" class="content">
            

  <div id="posts" class="posts-expand">
    

  

  
  
  

  <article class="post post-type-normal " itemscope itemtype="http://schema.org/Article">
    <link itemprop="mainEntityOfPage" href="http://www.colyfu.com/paper-express/paper-express-201705.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="name" content="Coly Fu">
      <meta itemprop="description" content="">
      <meta itemprop="image" content="http://onzjn6hm6.bkt.clouddn.com/avatar.jpg">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="Things worth spreading!">
    </span>

    
      <header class="post-header">

        
        
          <h2 class="post-title" itemprop="name headline">
            
            
              
                文献速递 (2017年5月)
              
            
          </h2>
        

        <div class="post-meta">
          <span class="post-time">
            
              <span class="post-meta-item-icon">
                <i class="fa fa-calendar-o"></i>
              </span>
              
                <span class="post-meta-item-text">发表于</span>
              
              <time title="创建于" itemprop="dateCreated datePublished" datetime="2017-06-03T18:36:01+08:00">
                2017-06-03
              </time>
            

            

            
          </span>

          
            <span class="post-category" >
            
              <span class="post-meta-divider">|</span>
            
              <span class="post-meta-item-icon">
                <i class="fa fa-folder-o"></i>
              </span>
              
                <span class="post-meta-item-text">分类于</span>
              
              
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/paper-express/" itemprop="url" rel="index">
                    <span itemprop="name">paper_express</span>
                  </a>
                </span>

                
                
              
            </span>
          

          
            
          

          
          
             <span id="/paper-express/paper-express-201705.html" class="leancloud_visitors" data-flag-title="文献速递 (2017年5月)">
               <span class="post-meta-divider">|</span>
               <span class="post-meta-item-icon">
                 <i class="fa fa-eye"></i>
               </span>
               
                 <span class="post-meta-item-text">阅读次数 </span>
               
                 <span class="leancloud-visitors-count"></span>
             </span>
          

          

          

          

        </div>
      </header>
    


    <div class="post-body" itemprop="articleBody">

      
      

      
        <h4 id="Two-extended-haplotype-blocks-are-associated-with-adaptation-to-high-altitude-habitats-in-East-African-honey-bees-gt-gt-gt"><a href="#Two-extended-haplotype-blocks-are-associated-with-adaptation-to-high-altitude-habitats-in-East-African-honey-bees-gt-gt-gt" class="headerlink" title="Two extended haplotype blocks are associated with adaptation to high altitude habitats in East African honey bees &gt;&gt;&gt;"></a>Two extended haplotype blocks are associated with adaptation to high altitude habitats in East African honey bees <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006792" target="_blank" rel="external">&gt;&gt;&gt;</a></h4><p>PLoS Genet 13(5): e1006792.<br>作者通过对东非意蜂高海拔和低海拔群体进行全基因组测序，发现高海拔和低海拔群体基因组之间存在两个遗传差异非常大的区段。其中一个区段存在一系列章鱼胺受体，推测与高海拔和低海拔群体的学习和觅食行为差异有关。这两个区段极大的遗传差异暗示了这些群体古老的起源，祖先群体很可能在传播到当前区域之前就存在不同的环境适应性了。</p>
<h4 id="Pollutants-and-Insecticides-Drive-Local-Adaptation-in-African-Malaria-Mosquitoes-gt-gt-gt"><a href="#Pollutants-and-Insecticides-Drive-Local-Adaptation-in-African-Malaria-Mosquitoes-gt-gt-gt" class="headerlink" title="Pollutants and Insecticides Drive Local Adaptation in African Malaria Mosquitoes &gt;&gt;&gt;"></a>Pollutants and Insecticides Drive Local Adaptation in African Malaria Mosquitoes <a href="https://academic.oup.com/mbe/article/34/5/1261/2996747/Pollutants-and-Insecticides-Drive-Local-Adaptation" target="_blank" rel="external">&gt;&gt;&gt;</a></h4><p>Mol Biol Evol (2017) 34 (5): 1261-1275.<br>对喀麦隆地区941只疟蚊进行简化基因组测序，得到8000个SNP标记。分析发现这些蚊子存在城市和乡村的群体遗传分化，城市污染地区的蚊子存在对杀虫剂的适应性。  </p>
<a id="more"></a>
<h4 id="Mitogenome-Diversity-in-Sardinians-A-Genetic-Window-onto-an-Island’s-Past-gt-gt-gt"><a href="#Mitogenome-Diversity-in-Sardinians-A-Genetic-Window-onto-an-Island’s-Past-gt-gt-gt" class="headerlink" title="Mitogenome Diversity in Sardinians: A Genetic Window onto an Island’s Past &gt;&gt;&gt;"></a>Mitogenome Diversity in Sardinians: A Genetic Window onto an Island’s Past <a href="https://academic.oup.com/mbe/article/34/5/1230/2977388/Mitogenome-Diversity-in-Sardinians-A-Genetic" target="_blank" rel="external">&gt;&gt;&gt;</a></h4><p>Mol Biol Evol (2017) 34 (5): 1230-1239.<br>撒丁岛人被认为是欧洲遗传组成的‘离群值’。作者对3,491个现代和21个古代撒丁岛人的线粒体基因组进行了研究，发现78.4%的现代线粒体基因组都能被分到89个单体型。作者发现大部分的线粒体基因组的溯祖时间大概在青铜时代，但也有一些稀有线粒体基因组的溯祖时间在新石器时代。  </p>
<h4 id="Evolution-of-Local-Mutation-Rate-and-Its-Determinants-gt-gt-gt"><a href="#Evolution-of-Local-Mutation-Rate-and-Its-Determinants-gt-gt-gt" class="headerlink" title="Evolution of Local Mutation Rate and Its Determinants &gt;&gt;&gt;"></a>Evolution of Local Mutation Rate and Its Determinants <a href="https://academic.oup.com/mbe/article/34/5/1100/2963012/Evolution-of-Local-Mutation-Rate-and-Its" target="_blank" rel="external">&gt;&gt;&gt;</a></h4><p>Mol Biol Evol (2017) 34 (5): 1100-1109.<br>人基因组不同的区域，突变率存在差异，部分可以通过DNA分子不同区域的性质来解释。但是不同物种的同源区域突变率的差异的形成原因还不清楚，本文给出了一些解释。</p>
<h4 id="The-Demographic-and-Adaptive-History-of-the-African-Green-Monkey-gt-gt-gt"><a href="#The-Demographic-and-Adaptive-History-of-the-African-Green-Monkey-gt-gt-gt" class="headerlink" title="The Demographic and Adaptive History of the African Green Monkey &gt;&gt;&gt;"></a>The Demographic and Adaptive History of the African Green Monkey <a href="https://academic.oup.com/mbe/article-abstract/34/5/1055/2983514/The-Demographic-and-Adaptive-History-of-the?redirectedFrom=fulltext" target="_blank" rel="external">&gt;&gt;&gt;</a></h4><p>Mol Biol Evol (2017) 34 (5): 1055-1065.</p>
<p>通过来自非洲绿猴所有5个群体的25个个体进行研究，揭示了绿猴的群体历史和适应性。</p>
<h4 id="GppFst-genomic-posterior-predictive-simulations-of-FST-and-dXY-for-identifying-outlier-loci-from-population-genomic-data-gt-gt-gt"><a href="#GppFst-genomic-posterior-predictive-simulations-of-FST-and-dXY-for-identifying-outlier-loci-from-population-genomic-data-gt-gt-gt" class="headerlink" title="GppFst: genomic posterior predictive simulations of FST and dXY for identifying outlier loci from population genomic data &gt;&gt;&gt;"></a>GppFst: genomic posterior predictive simulations of FST and dXY for identifying outlier loci from population genomic data <a href="https://academic.oup.com/bioinformatics/article-abstract/33/9/1414/2736363/GppFst-genomic-posterior-predictive-simulations-of?redirectedFrom=fulltext" target="_blank" rel="external">&gt;&gt;&gt;</a></h4><p>Bioinformatics (2017) 33 (9): 1414-1415.<br>通过后验预测模拟FST和dXY来发现基因组受选择区域。</p>
<h4 id="fastMitoCalc-an-ultra-fast-program-to-estimate-mitochondrial-DNA-copy-number-from-whole-genome-sequences-gt-gt-gt"><a href="#fastMitoCalc-an-ultra-fast-program-to-estimate-mitochondrial-DNA-copy-number-from-whole-genome-sequences-gt-gt-gt" class="headerlink" title="fastMitoCalc: an ultra-fast program to estimate mitochondrial DNA copy number from whole-genome sequences &gt;&gt;&gt;"></a>fastMitoCalc: an ultra-fast program to estimate mitochondrial DNA copy number from whole-genome sequences <a href="https://academic.oup.com/bioinformatics/article-abstract/33/9/1399/2832783/fastMitoCalc-an-ultra-fast-program-to-estimate?redirectedFrom=fulltext" target="_blank" rel="external">&gt;&gt;&gt;</a></h4><p>Bioinformatics (2017) 33 (9): 1399-1401.<br>快速估计细胞中平均线粒体拷贝数。其基本思想就是和常染色体测序深度比较。只是没有计算常染色体所有区域，只抽取了0.1%来计算，这样大大加快了计算速度。（这也能发Bioinformatics？？？）</p>
<h4 id="VCF-kit-assorted-utilities-for-the-variant-call-format-gt-gt-gt"><a href="#VCF-kit-assorted-utilities-for-the-variant-call-format-gt-gt-gt" class="headerlink" title="VCF-kit: assorted utilities for the variant call format &gt;&gt;&gt;"></a>VCF-kit: assorted utilities for the variant call format <a href="https://academic.oup.com/bioinformatics/article/33/10/1581/2908861/VCF-kit-assorted-utilities-for-the-variant-call" target="_blank" rel="external">&gt;&gt;&gt;</a></h4><p>Bioinformatics (2017) 33 (10): 1581-1582.<br>对VCF文件进行过滤等计算的工具。其中设计引物的功能可能是比较其他工具比较实用的。</p>
<h4 id="KNIME4NGS-a-comprehensive-toolbox-for-next-generation-sequencing-analysis-gt-gt-gt"><a href="#KNIME4NGS-a-comprehensive-toolbox-for-next-generation-sequencing-analysis-gt-gt-gt" class="headerlink" title="KNIME4NGS: a comprehensive toolbox for next generation sequencing analysis &gt;&gt;&gt;"></a>KNIME4NGS: a comprehensive toolbox for next generation sequencing analysis <a href="https://academic.oup.com/bioinformatics/article-abstract/33/10/1565/2871246/KNIME4NGS-a-comprehensive-toolbox-for-next?redirectedFrom=fulltext" target="_blank" rel="external">&gt;&gt;&gt;</a></h4><p>Bioinformatics (2017) 33 (10): 1565-1567.<br>NGS数据分析pipeline，包括RNA的差异表达，DNA的质控，比对，SNP calling，BQSR，BQSR，VEP等等工具。</p>
<h4 id="Widespread-Allelic-Heterogeneity-in-Complex-Traits-gt-gt-gt-30149-0"><a href="#Widespread-Allelic-Heterogeneity-in-Complex-Traits-gt-gt-gt-30149-0" class="headerlink" title="Widespread Allelic Heterogeneity in Complex Traits &gt;&gt;&gt;30149-0)"></a>Widespread Allelic Heterogeneity in Complex Traits <a href="http://www.cell.com/ajhg/abstract/S0002-9297(17" target="_blank" rel="external">&gt;&gt;&gt;</a>30149-0)</h4><p>AJHG (2017) 100(5): 789-802<br>等位基因异质性普遍存在于复杂性状中，这降低了统计的显著性。理解等位基因异质性，有助于我们开发新的方法进行遗传定位。  </p>
<h4 id="Inferring-Human-Demographic-Histories-of-Non-African-Populations-from-Patterns-of-Allele-Sharing-gt-gt-gt-30146-5"><a href="#Inferring-Human-Demographic-Histories-of-Non-African-Populations-from-Patterns-of-Allele-Sharing-gt-gt-gt-30146-5" class="headerlink" title="Inferring Human Demographic Histories of Non-African Populations from Patterns of Allele Sharing &gt;&gt;&gt;30146-5)"></a>Inferring Human Demographic Histories of Non-African Populations from Patterns of Allele Sharing <a href="http://www.cell.com/ajhg/abstract/S0002-9297(17" target="_blank" rel="external">&gt;&gt;&gt;</a>30146-5)</h4><p>AJHG (2017) 100(5): 766-772.<br>通过简单的基于简约法的分析揭示东亚人和美拉尼西亚人是姐妹群。</p>
<h4 id="Dispersals-and-genetic-adaptation-of-Bantu-speaking-populations-in-Africa-and-North-America-gt-gt-gt"><a href="#Dispersals-and-genetic-adaptation-of-Bantu-speaking-populations-in-Africa-and-North-America-gt-gt-gt" class="headerlink" title="Dispersals and genetic adaptation of Bantu-speaking populations in Africa and North America &gt;&gt;&gt;"></a>Dispersals and genetic adaptation of Bantu-speaking populations in Africa and North America <a href="http://science.sciencemag.org/content/356/6337/543" target="_blank" rel="external">&gt;&gt;&gt;</a></h4><p>Science  05 May 2017:<br>Vol. 356, Issue 6337, pp. 543-546<br>对来自35个群体的1318个班图语系人的遗传数据进行分析，发现在向非洲南部和东部扩散之前，班图语系人首先沿着赤道雨林向南迁徙。和当地人的混合，帮助了班图语系人适应当地环境。发现了非洲中西部班图语系人对非裔美国人的遗传贡献。  </p>
<h4 id="New-advances-in-sequence-assembly-gt-gt-gt"><a href="#New-advances-in-sequence-assembly-gt-gt-gt" class="headerlink" title="New advances in sequence assembly &gt;&gt;&gt;"></a>New advances in sequence assembly <a href="http://genome.cshlp.org/content/27/5/xi.full" target="_blank" rel="external">&gt;&gt;&gt;</a></h4><p>Genome Res. May 2017; 27 <a href="http://genome.cshlp.org/content/27/5.toc" target="_blank" rel="external">(5)</a><br>这篇文章是编辑对于本期涉及组装方法的综述。这一期的文章主要讲基因组组装方法的进展，特别是长读长reads的组装方法。</p>
<h4 id="Discovery-and-genotyping-of-structural-variation-from-long-read-haploid-genome-sequence-data-gt-gt-gt"><a href="#Discovery-and-genotyping-of-structural-variation-from-long-read-haploid-genome-sequence-data-gt-gt-gt" class="headerlink" title="Discovery and genotyping of structural variation from long-read haploid genome sequence data &gt;&gt;&gt;"></a>Discovery and genotyping of structural variation from long-read haploid genome sequence data <a href="http://genome.cshlp.org/content/27/5/677.abstract" target="_blank" rel="external">&gt;&gt;&gt;</a></h4><p>Genome Res. 2017. 27: 677-685<br>利用长读长reads检测结构变异。</p>
<h4 id="A-comparative-evaluation-of-genome-assembly-reconciliation-tools-gt-gt-gt"><a href="#A-comparative-evaluation-of-genome-assembly-reconciliation-tools-gt-gt-gt" class="headerlink" title="A comparative evaluation of genome assembly reconciliation tools &gt;&gt;&gt;"></a>A comparative evaluation of genome assembly reconciliation tools <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1213-3" target="_blank" rel="external">&gt;&gt;&gt;</a></h4><p>Genome Biology 2017 18:93<br>在基因组组装时，往往会用多个软件和参数组装多个版本，一般会挑一个最好的结果发表，也有将多个版本合并在一起，得到一个一致组装序列。对于后者，工具很多，本文对这些工具进行了评估。</p>
<h4 id="African-genomes-illuminate-the-early-history-and-transition-to-selfing-in-Arabidopsis-thaliana-gt-gt-gt"><a href="#African-genomes-illuminate-the-early-history-and-transition-to-selfing-in-Arabidopsis-thaliana-gt-gt-gt" class="headerlink" title="African genomes illuminate the early history and transition to selfing in Arabidopsis thaliana &gt;&gt;&gt;"></a>African genomes illuminate the early history and transition to selfing in Arabidopsis thaliana <a href="http://www.pnas.org/content/114/20/5213.full" target="_blank" rel="external">&gt;&gt;&gt;</a></h4><p>PNAS (2017) 114(20): 5213–5218</p>
<p>作者对78个非洲拟南芥进行测序，同1000多个已经测序的欧亚拟南芥一起分析。发现非洲拟南芥就是当地起源的，而且多态性最高，还发现拟南芥的自交性状是在非洲一个地方转变完成。文章对拟南芥的分歧时间，群体动态进行了推断。（研究植物群体历史的文章比较少，这篇看上去很有参考价值！）</p>
<h4 id="digit—a-tool-for-detection-and-identification-of-genomic-interchromosomal-translocations-gt-gt-gt"><a href="#digit—a-tool-for-detection-and-identification-of-genomic-interchromosomal-translocations-gt-gt-gt" class="headerlink" title="digit—a tool for detection and identification of genomic interchromosomal translocations &gt;&gt;&gt;"></a>digit—a tool for detection and identification of genomic interchromosomal translocations <a href="https://academic.oup.com/nar/article/45/9/e72/2962178/digit-a-tool-for-detection-and-identification-of" target="_blank" rel="external">&gt;&gt;&gt;</a></h4><p>Nucleic Acids Res (2017) 45 (9): e72.<br>基于pair end和mate pair数据检测染色体间异位的工具。</p>
<h4 id="Signatures-of-adaptation-in-the-weedy-rice-genome-gt-gt-gt"><a href="#Signatures-of-adaptation-in-the-weedy-rice-genome-gt-gt-gt" class="headerlink" title="Signatures of adaptation in the weedy rice genome &gt;&gt;&gt;"></a>Signatures of adaptation in the weedy rice genome <a href="http://www.nature.com/ng/journal/v49/n5/full/ng.3825.html" target="_blank" rel="external">&gt;&gt;&gt;</a></h4><p>Nature Genetics 49, 811–814 (2017)<br>作者通过全基因组测序研究了美国两种主要杂草稻的起源和适应性。杂草稻从栽培稻祖先去驯化在其进化历史上扮演关键作用。</p>
<h4 id="Transitioning-from-association-to-causation-with-eQTLs-gt-gt-gt"><a href="#Transitioning-from-association-to-causation-with-eQTLs-gt-gt-gt" class="headerlink" title="Transitioning from association to causation with eQTLs &gt;&gt;&gt;"></a>Transitioning from association to causation with eQTLs <a href="http://www.nature.com/nrg/journal/v18/n5/full/nrg.2017.22.html" target="_blank" rel="external">&gt;&gt;&gt;</a></h4><p>Nature Reviews Genetics 18, 271 (2017)<br>从关联到因果的转变－通过eQTLs。该文章介绍了AJHG的两篇文章，说明了eQTLs在寻找causal gene的作用。</p>
<h4 id="Sequencing-and-de-novo-assembly-of-a-near-complete-indica-rice-genome-gt-gt-gt"><a href="#Sequencing-and-de-novo-assembly-of-a-near-complete-indica-rice-genome-gt-gt-gt" class="headerlink" title="Sequencing and de novo assembly of a near complete indica rice genome &gt;&gt;&gt;"></a>Sequencing and de novo assembly of a near complete indica rice genome <a href="http://www.nature.com/articles/ncomms15324" target="_blank" rel="external">&gt;&gt;&gt;</a></h4><p>Nature Communications 8, 15324 (2017)<br>通过单分子测序，遗传图谱，fosmid等组装了籼稻Shuhui498的基因组，组装长度为390.3 Mb，覆盖Shuhui498基因组99%以上。</p>
<h4 id="An-integrated-model-for-detecting-significant-chromatin-interactions-from-high-resolution-Hi-C-data-gt-gt-gt"><a href="#An-integrated-model-for-detecting-significant-chromatin-interactions-from-high-resolution-Hi-C-data-gt-gt-gt" class="headerlink" title="An integrated model for detecting significant chromatin interactions from high-resolution Hi-C data &gt;&gt;&gt;"></a>An integrated model for detecting significant chromatin interactions from high-resolution Hi-C data <a href="http://www.nature.com/articles/ncomms15454" target="_blank" rel="external">&gt;&gt;&gt;</a></h4><p>Nature Communications 8, 15454 (2017)<br>通过高分辨率的Hi-C数据检测显著染色体互作的整合模型。</p>
<h4 id="Genomic-variation-associated-with-local-adaptation-of-weedy-rice-during-de-domestication-gt-gt-gt"><a href="#Genomic-variation-associated-with-local-adaptation-of-weedy-rice-during-de-domestication-gt-gt-gt" class="headerlink" title="Genomic variation associated with local adaptation of weedy rice during de-domestication &gt;&gt;&gt;"></a>Genomic variation associated with local adaptation of weedy rice during de-domestication <a href="http://www.nature.com/articles/ncomms15323" target="_blank" rel="external">&gt;&gt;&gt;</a></h4><p>Nature Communications 8, 15323 (2017)<br>作者对155株杂草稻和76株当地栽培稻进行了平均18.2 x的全基因组测序。通过系统发育树和群体动态分析发现，中国的杂草稻是独立从栽培稻去驯化形成的，中间经理了强的瓶颈效应。虽然有多个起源，杂草稻的关键基因显示了趋同进化。在杂草稻中鉴定出Mb大小的区段显示经历过平衡选择，而栽培稻没有。</p>

      
    </div>

    <div>
      
        

      
    </div>

    <div>
      
        
  <div style="padding: 10px 0; margin: 20px auto; width: 90%; text-align: center;">
    <div>如果我的博客能帮你节约时间，我会非常高兴你请我喝杯咖啡。</div>
    <button id="rewardButton" disable="enable" onclick="var qr = document.getElementById('QR'); if (qr.style.display === 'none') {qr.style.display='block';} else {qr.style.display='none'}">
      <span>赏</span>
    </button>
    <div id="QR" style="display: none;">
      
        <div id="wechat" style="display: inline-block">
          <img id="wechat_qr" src="http://onzjn6hm6.bkt.clouddn.com/weixin.jpg" alt="Coly Fu WeChat Pay"/>
          <p>微信打赏</p>
        </div>
      
      
        <div id="alipay" style="display: inline-block">
          <img id="alipay_qr" src="http://onzjn6hm6.bkt.clouddn.com/alipay.jpg" alt="Coly Fu Alipay"/>
          <p>支付宝打赏</p>
        </div>
      
    </div>
  </div>


      
    </div>

    <div>
      
        

      
    </div>

    <footer class="post-footer">
      
        <div class="post-tags">
          
            <a href="/tags/文献导读/" rel="tag"># 文献导读</a>
          
        </div>
      

      
        
      

      
        <div class="post-nav">
          <div class="post-nav-next post-nav-item">
            
              <a href="/genomics/liftover-chain.html" rel="next" title="LiftOver chain 文件创建流程">
                <i class="fa fa-chevron-left"></i> LiftOver chain 文件创建流程
              </a>
            
          </div>

          <span class="post-nav-divider"></span>

          <div class="post-nav-prev post-nav-item">
            
              <a href="/programming/blockread-python.html" rel="prev" title="python按指定分隔符读取文件">
                python按指定分隔符读取文件 <i class="fa fa-chevron-right"></i>
              </a>
            
          </div>
        </div>
      

      
      
    </footer>
  </article>



    <div class="post-spread">
      
    </div>
  </div>


          </div>
          


          
  <div class="comments" id="comments">
    
      <div id="lv-container" data-id="city" data-uid="MTAyMC8yODA3OC80NjUx"></div>
    
  </div>


        </div>
        
          
  
  <div class="sidebar-toggle">
    <div class="sidebar-toggle-line-wrap">
      <span class="sidebar-toggle-line sidebar-toggle-line-first"></span>
      <span class="sidebar-toggle-line sidebar-toggle-line-middle"></span>
      <span class="sidebar-toggle-line sidebar-toggle-line-last"></span>
    </div>
  </div>

  <aside id="sidebar" class="sidebar">
    <div class="sidebar-inner">

      

      
        <ul class="sidebar-nav motion-element">
          <li class="sidebar-nav-toc sidebar-nav-active" data-target="post-toc-wrap" >
            文章目录
          </li>
          <li class="sidebar-nav-overview" data-target="site-overview">
            站点概览
          </li>
        </ul>
      

      <section class="site-overview sidebar-panel">
        <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
          <img class="site-author-image" itemprop="image"
               src="http://onzjn6hm6.bkt.clouddn.com/avatar.jpg"
               alt="Coly Fu" />
          <p class="site-author-name" itemprop="name">Coly Fu</p>
           
              <p class="site-description motion-element" itemprop="description"></p>
           
        </div>
        <nav class="site-state motion-element">

          
            <div class="site-state-item site-state-posts">
              <a href="/archives">
                <span class="site-state-item-count">16</span>
                <span class="site-state-item-name">日志</span>
              </a>
            </div>
          

          
            
            
            <div class="site-state-item site-state-categories">
              <a href="/categories/index.html">
                <span class="site-state-item-count">6</span>
                <span class="site-state-item-name">分类</span>
              </a>
            </div>
          

          
            
            
            <div class="site-state-item site-state-tags">
              <a href="/tags/index.html">
                <span class="site-state-item-count">33</span>
                <span class="site-state-item-name">标签</span>
              </a>
            </div>
          

        </nav>

        
          <div class="feed-link motion-element">
            <a href="/atom.xml" rel="alternate">
              <i class="fa fa-rss"></i>
              RSS
            </a>
          </div>
        

        <div class="links-of-author motion-element">
          
        </div>

        
        

        
        

        


      </section>

      
      <!--noindex-->
        <section class="post-toc-wrap motion-element sidebar-panel sidebar-panel-active">
          <div class="post-toc">

            
              
            

            
              <div class="post-toc-content"><ol class="nav"><li class="nav-item nav-level-4"><a class="nav-link" href="#Two-extended-haplotype-blocks-are-associated-with-adaptation-to-high-altitude-habitats-in-East-African-honey-bees-gt-gt-gt"><span class="nav-number">1.</span> <span class="nav-text">Two extended haplotype blocks are associated with adaptation to high altitude habitats in East African honey bees >>></span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Pollutants-and-Insecticides-Drive-Local-Adaptation-in-African-Malaria-Mosquitoes-gt-gt-gt"><span class="nav-number">2.</span> <span class="nav-text">Pollutants and Insecticides Drive Local Adaptation in African Malaria Mosquitoes >>></span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Mitogenome-Diversity-in-Sardinians-A-Genetic-Window-onto-an-Island’s-Past-gt-gt-gt"><span class="nav-number">3.</span> <span class="nav-text">Mitogenome Diversity in Sardinians: A Genetic Window onto an Island’s Past >>></span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Evolution-of-Local-Mutation-Rate-and-Its-Determinants-gt-gt-gt"><span class="nav-number">4.</span> <span class="nav-text">Evolution of Local Mutation Rate and Its Determinants >>></span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#The-Demographic-and-Adaptive-History-of-the-African-Green-Monkey-gt-gt-gt"><span class="nav-number">5.</span> <span class="nav-text">The Demographic and Adaptive History of the African Green Monkey >>></span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#GppFst-genomic-posterior-predictive-simulations-of-FST-and-dXY-for-identifying-outlier-loci-from-population-genomic-data-gt-gt-gt"><span class="nav-number">6.</span> <span class="nav-text">GppFst: genomic posterior predictive simulations of FST and dXY for identifying outlier loci from population genomic data >>></span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#fastMitoCalc-an-ultra-fast-program-to-estimate-mitochondrial-DNA-copy-number-from-whole-genome-sequences-gt-gt-gt"><span class="nav-number">7.</span> <span class="nav-text">fastMitoCalc: an ultra-fast program to estimate mitochondrial DNA copy number from whole-genome sequences >>></span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#VCF-kit-assorted-utilities-for-the-variant-call-format-gt-gt-gt"><span class="nav-number">8.</span> <span class="nav-text">VCF-kit: assorted utilities for the variant call format >>></span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#KNIME4NGS-a-comprehensive-toolbox-for-next-generation-sequencing-analysis-gt-gt-gt"><span class="nav-number">9.</span> <span class="nav-text">KNIME4NGS: a comprehensive toolbox for next generation sequencing analysis >>></span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Widespread-Allelic-Heterogeneity-in-Complex-Traits-gt-gt-gt-30149-0"><span class="nav-number">10.</span> <span class="nav-text">Widespread Allelic Heterogeneity in Complex Traits >>>30149-0)</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Inferring-Human-Demographic-Histories-of-Non-African-Populations-from-Patterns-of-Allele-Sharing-gt-gt-gt-30146-5"><span class="nav-number">11.</span> <span class="nav-text">Inferring Human Demographic Histories of Non-African Populations from Patterns of Allele Sharing >>>30146-5)</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Dispersals-and-genetic-adaptation-of-Bantu-speaking-populations-in-Africa-and-North-America-gt-gt-gt"><span class="nav-number">12.</span> <span class="nav-text">Dispersals and genetic adaptation of Bantu-speaking populations in Africa and North America >>></span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#New-advances-in-sequence-assembly-gt-gt-gt"><span class="nav-number">13.</span> <span class="nav-text">New advances in sequence assembly >>></span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Discovery-and-genotyping-of-structural-variation-from-long-read-haploid-genome-sequence-data-gt-gt-gt"><span class="nav-number">14.</span> <span class="nav-text">Discovery and genotyping of structural variation from long-read haploid genome sequence data >>></span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#A-comparative-evaluation-of-genome-assembly-reconciliation-tools-gt-gt-gt"><span class="nav-number">15.</span> <span class="nav-text">A comparative evaluation of genome assembly reconciliation tools >>></span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#African-genomes-illuminate-the-early-history-and-transition-to-selfing-in-Arabidopsis-thaliana-gt-gt-gt"><span class="nav-number">16.</span> <span class="nav-text">African genomes illuminate the early history and transition to selfing in Arabidopsis thaliana >>></span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#digit—a-tool-for-detection-and-identification-of-genomic-interchromosomal-translocations-gt-gt-gt"><span class="nav-number">17.</span> <span class="nav-text">digit—a tool for detection and identification of genomic interchromosomal translocations >>></span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Signatures-of-adaptation-in-the-weedy-rice-genome-gt-gt-gt"><span class="nav-number">18.</span> <span class="nav-text">Signatures of adaptation in the weedy rice genome >>></span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Transitioning-from-association-to-causation-with-eQTLs-gt-gt-gt"><span class="nav-number">19.</span> <span class="nav-text">Transitioning from association to causation with eQTLs >>></span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Sequencing-and-de-novo-assembly-of-a-near-complete-indica-rice-genome-gt-gt-gt"><span class="nav-number">20.</span> <span class="nav-text">Sequencing and de novo assembly of a near complete indica rice genome >>></span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#An-integrated-model-for-detecting-significant-chromatin-interactions-from-high-resolution-Hi-C-data-gt-gt-gt"><span class="nav-number">21.</span> <span class="nav-text">An integrated model for detecting significant chromatin interactions from high-resolution Hi-C data >>></span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Genomic-variation-associated-with-local-adaptation-of-weedy-rice-during-de-domestication-gt-gt-gt"><span class="nav-number">22.</span> <span class="nav-text">Genomic variation associated with local adaptation of weedy rice during de-domestication >>></span></a></li></ol></div>
            

          </div>
        </section>
      <!--/noindex-->
      

      

    </div>
  </aside>


        
      </div>
    </main>

    <footer id="footer" class="footer">
      <div class="footer-inner">
        <div class="copyright" >
  
  &copy;  2016 - 
  <span itemprop="copyrightYear">2018</span>
  <span class="with-love">
    <i class="fa fa-heart"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">Coly Fu</span>
</div>



        

        
      </div>
    </footer>

    
      <div class="back-to-top">
        <i class="fa fa-arrow-up"></i>
        
      </div>
    

  </div>

  

<script type="text/javascript">
  if (Object.prototype.toString.call(window.Promise) !== '[object Function]') {
    window.Promise = null;
  }
</script>









  






  
  <script type="text/javascript" src="/lib/jquery/index.js?v=2.1.3"></script>

  
  <script type="text/javascript" src="/lib/fastclick/lib/fastclick.min.js?v=1.0.6"></script>

  
  <script type="text/javascript" src="/lib/jquery_lazyload/jquery.lazyload.js?v=1.9.7"></script>

  
  <script type="text/javascript" src="/lib/velocity/velocity.min.js?v=1.2.1"></script>

  
  <script type="text/javascript" src="/lib/velocity/velocity.ui.min.js?v=1.2.1"></script>

  
  <script type="text/javascript" src="/lib/fancybox/source/jquery.fancybox.pack.js?v=2.1.5"></script>


  


  <script type="text/javascript" src="/js/src/utils.js?v=5.1.0"></script>

  <script type="text/javascript" src="/js/src/motion.js?v=5.1.0"></script>



  
  

  
  <script type="text/javascript" src="/js/src/scrollspy.js?v=5.1.0"></script>
<script type="text/javascript" src="/js/src/post-details.js?v=5.1.0"></script>



  


  <script type="text/javascript" src="/js/src/bootstrap.js?v=5.1.0"></script>



  


  




	





  





  





  
    <script type="text/javascript">
      (function(d, s) {
        var j, e = d.getElementsByTagName(s)[0];
        if (typeof LivereTower === 'function') { return; }
        j = d.createElement(s);
        j.src = 'https://cdn-city.livere.com/js/embed.dist.js';
        j.async = true;
        e.parentNode.insertBefore(j, e);
      })(document, 'script');
    </script>
  






  

  <script type="text/javascript">
    // Popup Window;
    var isfetched = false;
    // Search DB path;
    var search_path = "search.xml";
    if (search_path.length == 0) {
      search_path = "search.xml";
    }
    var path = "/" + search_path;
    // monitor main search box;

    function proceedsearch() {
      $("body")
        .append('<div class="search-popup-overlay local-search-pop-overlay"></div>')
        .css('overflow', 'hidden');
      $('.popup').toggle();
    }
    // search function;
    var searchFunc = function(path, search_id, content_id) {
      'use strict';
      $.ajax({
        url: path,
        dataType: "xml",
        async: true,
        success: function( xmlResponse ) {
          // get the contents from search data
          isfetched = true;
          $('.popup').detach().appendTo('.header-inner');
          var datas = $( "entry", xmlResponse ).map(function() {
            return {
              title: $( "title", this ).text(),
              content: $("content",this).text(),
              url: $( "url" , this).text()
            };
          }).get();
          var $input = document.getElementById(search_id);
          var $resultContent = document.getElementById(content_id);
          $input.addEventListener('input', function(){
            var matchcounts = 0;
            var str='<ul class=\"search-result-list\">';
            var keywords = this.value.trim().toLowerCase().split(/[\s\-]+/);
            $resultContent.innerHTML = "";
            if (this.value.trim().length > 1) {
              // perform local searching
              datas.forEach(function(data) {
                var isMatch = false;
                var content_index = [];
                var data_title = data.title.trim().toLowerCase();
                var data_content = data.content.trim().replace(/<[^>]+>/g,"").toLowerCase();
                var data_url = decodeURIComponent(data.url);
                var index_title = -1;
                var index_content = -1;
                var first_occur = -1;
                // only match artiles with not empty titles and contents
                if(data_title != '') {
                  keywords.forEach(function(keyword, i) {
                    index_title = data_title.indexOf(keyword);
                    index_content = data_content.indexOf(keyword);
                    if( index_title >= 0 || index_content >= 0 ){
                      isMatch = true;
                      if (i == 0) {
                        first_occur = index_content;
                      }
                    }

                  });
                }
                // show search results
                if (isMatch) {
                  matchcounts += 1;
                  str += "<li><a href='"+ data_url +"' class='search-result-title'>"+ data_title +"</a>";
                  var content = data.content.trim().replace(/<[^>]+>/g,"");
                  if (first_occur >= 0) {
                    // cut out 100 characters
                    var start = first_occur - 20;
                    var end = first_occur + 80;
                    if(start < 0){
                      start = 0;
                    }
                    if(start == 0){
                      end = 50;
                    }
                    if(end > content.length){
                      end = content.length;
                    }
                    var match_content = content.substring(start, end);
                    // highlight all keywords
                    keywords.forEach(function(keyword){
                      var regS = new RegExp(keyword, "gi");
                      match_content = match_content.replace(regS, "<b class=\"search-keyword\">"+keyword+"</b>");
                    });

                    str += "<p class=\"search-result\">" + match_content +"...</p>"
                  }
                  str += "</li>";
                }
              })};
            str += "</ul>";
            if (matchcounts == 0) { str = '<div id="no-result"><i class="fa fa-frown-o fa-5x" /></div>' }
            if (keywords == "") { str = '<div id="no-result"><i class="fa fa-search fa-5x" /></div>' }
            $resultContent.innerHTML = str;
          });
          proceedsearch();
        }
      });}

    // handle and trigger popup window;
    $('.popup-trigger').click(function(e) {
      e.stopPropagation();
      if (isfetched == false) {
        searchFunc(path, 'local-search-input', 'local-search-result');
      } else {
        proceedsearch();
      };
    });

    $('.popup-btn-close').click(function(e){
      $('.popup').hide();
      $(".local-search-pop-overlay").remove();
      $('body').css('overflow', '');
    });
    $('.popup').click(function(e){
      e.stopPropagation();
    });
  </script>





  

  
  <script src="https://cdn1.lncld.net/static/js/av-core-mini-0.6.1.js"></script>
  <script>AV.initialize("TYEr3gWwGEom2g4BHIufivSY-gzGzoHsz", "76KRaa6vXccIYhlGecUXpv6W");</script>
  <script>
    function showTime(Counter) {
      var query = new AV.Query(Counter);
      var entries = [];
      var $visitors = $(".leancloud_visitors");

      $visitors.each(function () {
        entries.push( $(this).attr("id").trim() );
      });

      query.containedIn('url', entries);
      query.find()
        .done(function (results) {
          var COUNT_CONTAINER_REF = '.leancloud-visitors-count';

          if (results.length === 0) {
            $visitors.find(COUNT_CONTAINER_REF).text(0);
            return;
          }

          for (var i = 0; i < results.length; i++) {
            var item = results[i];
            var url = item.get('url');
            var time = item.get('time');
            var element = document.getElementById(url);

            $(element).find(COUNT_CONTAINER_REF).text(time);
          }
          for(var i = 0; i < entries.length; i++) {
            var url = entries[i];
            var element = document.getElementById(url);
            var countSpan = $(element).find(COUNT_CONTAINER_REF);
            if( countSpan.text() == '') {
              countSpan.text(0);
            }
          }
        })
        .fail(function (object, error) {
          console.log("Error: " + error.code + " " + error.message);
        });
    }

    function addCount(Counter) {
      var $visitors = $(".leancloud_visitors");
      var url = $visitors.attr('id').trim();
      var title = $visitors.attr('data-flag-title').trim();
      var query = new AV.Query(Counter);

      query.equalTo("url", url);
      query.find({
        success: function(results) {
          if (results.length > 0) {
            var counter = results[0];
            counter.fetchWhenSave(true);
            counter.increment("time");
            counter.save(null, {
              success: function(counter) {
                var $element = $(document.getElementById(url));
                $element.find('.leancloud-visitors-count').text(counter.get('time'));
              },
              error: function(counter, error) {
                console.log('Failed to save Visitor num, with error message: ' + error.message);
              }
            });
          } else {
            var newcounter = new Counter();
            /* Set ACL */
            var acl = new AV.ACL();
            acl.setPublicReadAccess(true);
            acl.setPublicWriteAccess(true);
            newcounter.setACL(acl);
            /* End Set ACL */
            newcounter.set("title", title);
            newcounter.set("url", url);
            newcounter.set("time", 1);
            newcounter.save(null, {
              success: function(newcounter) {
                var $element = $(document.getElementById(url));
                $element.find('.leancloud-visitors-count').text(newcounter.get('time'));
              },
              error: function(newcounter, error) {
                console.log('Failed to create');
              }
            });
          }
        },
        error: function(error) {
          console.log('Error:' + error.code + " " + error.message);
        }
      });
    }

    $(function() {
      var Counter = AV.Object.extend("Counter");
      if ($('.leancloud_visitors').length == 1) {
        addCount(Counter);
      } else if ($('.post-title-link').length > 1) {
        showTime(Counter);
      }
    });
  </script>



  
<script>
(function(){
    var bp = document.createElement('script');
    var curProtocol = window.location.protocol.split(':')[0];
    if (curProtocol === 'https') {
        bp.src = 'https://zz.bdstatic.com/linksubmit/push.js';        
    }
    else {
        bp.src = 'http://push.zhanzhang.baidu.com/push.js';
    }
    var s = document.getElementsByTagName("script")[0];
    s.parentNode.insertBefore(bp, s);
})();
</script>


  

  

</body>
</html>
