<!doctype html>



  


<html class="theme-next mist use-motion" lang="zh-Hans">
<head>
  <meta charset="UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=edge" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1"/>



<meta http-equiv="Cache-Control" content="no-transform" />
<meta http-equiv="Cache-Control" content="no-siteapp" />















  
  
  <link href="/lib/fancybox/source/jquery.fancybox.css?v=2.1.5" rel="stylesheet" type="text/css" />




  
  
  
  

  
    
    
  

  

  

  

  

  
    
    
    <link href="//fonts.googleapis.com/css?family=Lato:300,300italic,400,400italic,700,700italic&subset=latin,latin-ext" rel="stylesheet" type="text/css">
  






<link href="/lib/font-awesome/css/font-awesome.min.css?v=4.6.2" rel="stylesheet" type="text/css" />

<link href="/css/main.css?v=5.1.0" rel="stylesheet" type="text/css" />


  <meta name="keywords" content="文献导读," />





  <link rel="alternate" href="/atom.xml" title="Things worth spreading!" type="application/atom+xml" />




  <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico?v=5.1.0" />






<meta name="description" content="The Rice Paradox: Multiple Origins but Single Domestication in Asian RiceMol Biol Evol (2017) 34 (4): 969-979.通过对几种栽培水稻和野生祖先的基因组比较，发现不同栽培稻亚种分别来自不同的野生祖先，但是只经历了一次驯化事件。最先被驯化的是O. sativa ssp. japonica，与O.">
<meta name="keywords" content="文献导读">
<meta property="og:type" content="article">
<meta property="og:title" content="文献速递 (2017年4月)">
<meta property="og:url" content="http://www.colyfu.com/paper-express/paper-express-201704.html">
<meta property="og:site_name" content="Things worth spreading!">
<meta property="og:description" content="The Rice Paradox: Multiple Origins but Single Domestication in Asian RiceMol Biol Evol (2017) 34 (4): 969-979.通过对几种栽培水稻和野生祖先的基因组比较，发现不同栽培稻亚种分别来自不同的野生祖先，但是只经历了一次驯化事件。最先被驯化的是O. sativa ssp. japonica，与O.">
<meta property="og:updated_time" content="2017-05-27T11:53:55.000Z">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="文献速递 (2017年4月)">
<meta name="twitter:description" content="The Rice Paradox: Multiple Origins but Single Domestication in Asian RiceMol Biol Evol (2017) 34 (4): 969-979.通过对几种栽培水稻和野生祖先的基因组比较，发现不同栽培稻亚种分别来自不同的野生祖先，但是只经历了一次驯化事件。最先被驯化的是O. sativa ssp. japonica，与O.">



<script type="text/javascript" id="hexo.configurations">
  var NexT = window.NexT || {};
  var CONFIG = {
    root: '/',
    scheme: 'Mist',
    sidebar: {"position":"left","display":"post","offset":12,"offset_float":0,"b2t":false,"scrollpercent":false},
    fancybox: true,
    motion: true,
    duoshuo: {
      userId: '0',
      author: '博主'
    },
    algolia: {
      applicationID: '',
      apiKey: '',
      indexName: '',
      hits: {"per_page":10},
      labels: {"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}
    }
  };
</script>



  <link rel="canonical" href="http://www.colyfu.com/paper-express/paper-express-201704.html"/>





  <title> 文献速递 (2017年4月) | Things worth spreading! </title>
</head>

<body itemscope itemtype="http://schema.org/WebPage" lang="zh-Hans">

  





  <script type="text/javascript">
    var _hmt = _hmt || [];
    (function() {
      var hm = document.createElement("script");
      hm.src = "https://hm.baidu.com/hm.js?da55ad3ecaab6360dc40a9dc42aaa258";
      var s = document.getElementsByTagName("script")[0];
      s.parentNode.insertBefore(hm, s);
    })();
  </script>










  
  
    
  

  <div class="container sidebar-position-left page-post-detail ">
    <div class="headband"></div>

    <header id="header" class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-wrapper">
  <div class="site-meta ">
    

    <div class="custom-logo-site-title">
      <a href="/"  class="brand" rel="start">
        <span class="logo-line-before"><i></i></span>
        <span class="site-title">Things worth spreading!</span>
        <span class="logo-line-after"><i></i></span>
      </a>
    </div>
      
        <h1 class="site-subtitle" itemprop="description">Knowledge is a good thing, and good things are worth spreading!</h1>
      
  </div>

  <div class="site-nav-toggle">
    <button>
      <span class="btn-bar"></span>
      <span class="btn-bar"></span>
      <span class="btn-bar"></span>
    </button>
  </div>
</div>

<nav class="site-nav">
  

  
    <ul id="menu" class="menu">
      
        
        <li class="menu-item menu-item-home">
          <a href="/" rel="section">
            
              <i class="menu-item-icon fa fa-fw fa-home"></i> <br />
            
            首页
          </a>
        </li>
      
        
        <li class="menu-item menu-item-categories">
          <a href="/categories" rel="section">
            
              <i class="menu-item-icon fa fa-fw fa-th"></i> <br />
            
            分类
          </a>
        </li>
      
        
        <li class="menu-item menu-item-about">
          <a href="/about" rel="section">
            
              <i class="menu-item-icon fa fa-fw fa-user"></i> <br />
            
            关于
          </a>
        </li>
      
        
        <li class="menu-item menu-item-archives">
          <a href="/archives" rel="section">
            
              <i class="menu-item-icon fa fa-fw fa-archive"></i> <br />
            
            归档
          </a>
        </li>
      
        
        <li class="menu-item menu-item-tags">
          <a href="/tags" rel="section">
            
              <i class="menu-item-icon fa fa-fw fa-tags"></i> <br />
            
            标签
          </a>
        </li>
      

      
        <li class="menu-item menu-item-search">
          
            <a href="javascript:;" class="popup-trigger">
          
            
              <i class="menu-item-icon fa fa-search fa-fw"></i> <br />
            
            搜索
          </a>
        </li>
      
    </ul>
  

  
    <div class="site-search">
      
  <div class="popup search-popup local-search-popup">
  <div class="local-search-header clearfix">
    <span class="search-icon">
      <i class="fa fa-search"></i>
    </span>
    <span class="popup-btn-close">
      <i class="fa fa-times-circle"></i>
    </span>
    <div class="local-search-input-wrapper">
      <input autocapitalize="off" autocomplete="off" autocorrect="off"
             placeholder="搜索..." spellcheck="false"
             type="text" id="local-search-input">
    </div>
  </div>
  <div id="local-search-result"></div>
</div>



    </div>
  
</nav>



 </div>
    </header>

    <main id="main" class="main">
      <div class="main-inner">
        <div class="content-wrap">
          <div id="content" class="content">
            

  <div id="posts" class="posts-expand">
    

  

  
  
  

  <article class="post post-type-normal " itemscope itemtype="http://schema.org/Article">
    <link itemprop="mainEntityOfPage" href="http://www.colyfu.com/paper-express/paper-express-201704.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="name" content="Coly Fu">
      <meta itemprop="description" content="">
      <meta itemprop="image" content="http://onzjn6hm6.bkt.clouddn.com/avatar.jpg">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="Things worth spreading!">
    </span>

    
      <header class="post-header">

        
        
          <h2 class="post-title" itemprop="name headline">
            
            
              
                文献速递 (2017年4月)
              
            
          </h2>
        

        <div class="post-meta">
          <span class="post-time">
            
              <span class="post-meta-item-icon">
                <i class="fa fa-calendar-o"></i>
              </span>
              
                <span class="post-meta-item-text">发表于</span>
              
              <time title="创建于" itemprop="dateCreated datePublished" datetime="2017-05-02T22:16:47+08:00">
                2017-05-02
              </time>
            

            

            
          </span>

          
            <span class="post-category" >
            
              <span class="post-meta-divider">|</span>
            
              <span class="post-meta-item-icon">
                <i class="fa fa-folder-o"></i>
              </span>
              
                <span class="post-meta-item-text">分类于</span>
              
              
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/paper-express/" itemprop="url" rel="index">
                    <span itemprop="name">paper_express</span>
                  </a>
                </span>

                
                
              
            </span>
          

          
            
          

          
          
             <span id="/paper-express/paper-express-201704.html" class="leancloud_visitors" data-flag-title="文献速递 (2017年4月)">
               <span class="post-meta-divider">|</span>
               <span class="post-meta-item-icon">
                 <i class="fa fa-eye"></i>
               </span>
               
                 <span class="post-meta-item-text">阅读次数 </span>
               
                 <span class="leancloud-visitors-count"></span>
             </span>
          

          

          

          

        </div>
      </header>
    


    <div class="post-body" itemprop="articleBody">

      
      

      
        <h4 id="The-Rice-Paradox-Multiple-Origins-but-Single-Domestication-in-Asian-Rice"><a href="#The-Rice-Paradox-Multiple-Origins-but-Single-Domestication-in-Asian-Rice" class="headerlink" title="The Rice Paradox: Multiple Origins but Single Domestication in Asian Rice"></a><a href="https://academic.oup.com/mbe/article/34/4/969/2897272/The-Rice-Paradox-Multiple-Origins-but-Single" target="_blank" rel="external">The Rice Paradox: Multiple Origins but Single Domestication in Asian Rice</a></h4><p>Mol Biol Evol (2017) 34 (4): 969-979.<br>通过对几种栽培水稻和野生祖先的基因组比较，发现不同栽培稻亚种分别来自不同的野生祖先，但是只经历了一次驯化事件。<br>最先被驯化的是<em>O. sativa</em> ssp. japonica，与<em>O. rufipogon</em>大概在∼13.1–24.1 ka前分歧。japonica与indica和aus的驯化前体存在基因流，导致后两者获得了驯化基因型。</p>
<h4 id="Genome-Sequencing-Reveals-the-Origin-of-the-Allotetraploid-Arabidopsis-suecica"><a href="#Genome-Sequencing-Reveals-the-Origin-of-the-Allotetraploid-Arabidopsis-suecica" class="headerlink" title="Genome Sequencing Reveals the Origin of the Allotetraploid Arabidopsis suecica"></a><a href="https://academic.oup.com/mbe/article/34/4/957/2838775/Genome-Sequencing-Reveals-the-Origin-of-the" target="_blank" rel="external">Genome Sequencing Reveals the Origin of the Allotetraploid Arabidopsis suecica</a></h4><p>Mol Biol Evol (2017) 34 (4): 957-968.<br>对15个异源四倍体拟南芥<em>Arabidopsis suecica</em>进行了重测序。发现其与多个祖先物种共享多态性，排除了单一起源。该物种大概在末次盛冰期(LGM)的东欧或欧亚大陆中部出现。还发现两个自交不亲和基因都没有功能。</p>
<a id="more"></a>
<h4 id="Deleterious-Variants-in-Asian-Rice-and-the-Potential-Cost-of-Domestication"><a href="#Deleterious-Variants-in-Asian-Rice-and-the-Potential-Cost-of-Domestication" class="headerlink" title="Deleterious Variants in Asian Rice and the Potential Cost of Domestication"></a><a href="https://academic.oup.com/mbe/article-abstract/34/4/908/2897207/Deleterious-Variants-in-Asian-Rice-and-the?redirectedFrom=fulltext" target="_blank" rel="external">Deleterious Variants in Asian Rice and the Potential Cost of Domestication</a></h4><p>Mol Biol Evol (2017) 34 (4): 908-924.<br>通过与野生种的比较，发现驯化种积累了更多(∼3–4%)的有害SNPs。这些有害SNPs 主要集中在低重组率的区域。在受选择区域，有害SNPs的频率增加和同义SNPs相近。驯化导致水稻从异交向以自交为主转变。</p>
<h4 id="A-Working-Model-of-the-Deep-Relationships-of-Diverse-Modern-Human-Genetic-Lineages-Outside-of-Africa"><a href="#A-Working-Model-of-the-Deep-Relationships-of-Diverse-Modern-Human-Genetic-Lineages-Outside-of-Africa" class="headerlink" title="A Working Model of the Deep Relationships of Diverse Modern Human Genetic Lineages Outside of Africa "></a><a href="https://academic.oup.com/mbe/article/34/4/889/2838774/A-Working-Model-of-the-Deep-Relationships-of" target="_blank" rel="external">A Working Model of the Deep Relationships of Diverse Modern Human Genetic Lineages Outside of Africa </a></h4><p>Mol Biol Evol (2017) 34 (4): 889-902.<br>人类走出非洲后，目前的遗传结构是怎样建立的，一直是人类历史研究的主要兴趣点。<br>本文作者提出了一个模型，能够很好的拟合各人群体（东亚，澳大利亚，美国本土，古欧亚大陆西部和北部）的基因型频率数据。</p>
<h4 id="What-is-adaptation-by-natural-selection-Perspectives-of-an-experimental-microbiologist"><a href="#What-is-adaptation-by-natural-selection-Perspectives-of-an-experimental-microbiologist" class="headerlink" title="What is adaptation by natural selection? Perspectives of an experimental microbiologist"></a><a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006668" target="_blank" rel="external">What is adaptation by natural selection? Perspectives of an experimental microbiologist</a></h4><p>PLoS Genet 13(4): e1006668.<br>微生物对自然选择适应的综述。微生物实验进化学。</p>
<h4 id="Evolutionary-history-of-Tibetans-inferred-from-whole-genome-sequencing"><a href="#Evolutionary-history-of-Tibetans-inferred-from-whole-genome-sequencing" class="headerlink" title="Evolutionary history of Tibetans inferred from whole-genome sequencing"></a><a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006675" target="_blank" rel="external">Evolutionary history of Tibetans inferred from whole-genome sequencing</a></h4><p> PLoS Genet 13(4): e1006675.<br> 通过对27个藏族人的全基因组测序，揭示其群体历史和受选择基因。</p>
<h4 id="The-time-and-place-of-European-admixture-in-Ashkenazi-Jewish-history"><a href="#The-time-and-place-of-European-admixture-in-Ashkenazi-Jewish-history" class="headerlink" title="The time and place of European admixture in Ashkenazi Jewish history"></a><a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006644" target="_blank" rel="external">The time and place of European admixture in Ashkenazi Jewish history</a></h4><p>PLoS Genet 13(4): e1006644.<br>推断德系犹太人的历史。用了一些新的推断方法。</p>
<h4 id="RECKONER-read-error-corrector-based-on-KMC"><a href="#RECKONER-read-error-corrector-based-on-KMC" class="headerlink" title="RECKONER: read error corrector based on KMC"></a><a href="https://academic.oup.com/bioinformatics/article-abstract/33/7/1086/2843893/RECKONER-read-error-corrector-based-on-KMC?redirectedFrom=fulltext" target="_blank" rel="external">RECKONER: read error corrector based on KMC</a></h4><p>Bioinformatics (2017) 33 (7): 1086-1089.<br>对测序的reads质量进行矫正。</p>
<h4 id="SVScore-an-impact-prediction-tool-for-structural-variation"><a href="#SVScore-an-impact-prediction-tool-for-structural-variation" class="headerlink" title="SVScore: an impact prediction tool for structural variation"></a><a href="https://academic.oup.com/bioinformatics/article/33/7/1083/2748212/SVScore-an-impact-prediction-tool-for-structural" target="_blank" rel="external">SVScore: an impact prediction tool for structural variation</a></h4><p>Bioinformatics (2017) 33 (7): 1083-1085.<br>对SV的影响进行预测。发现高影响的SV在群体中的频率更低，说明它们受到纯化选择。SVscore对预测有害突变的效果比其他软件好。研究者还发现duplication比deletion受到更强的选择。</p>
<h4 id="Fast-and-accurate-phylogeny-reconstruction-using-filtered-spaced-word-matches"><a href="#Fast-and-accurate-phylogeny-reconstruction-using-filtered-spaced-word-matches" class="headerlink" title="Fast and accurate phylogeny reconstruction using filtered spaced-word matches"></a><a href="https://academic.oup.com/bioinformatics/article/33/7/971/2883388/Fast-and-accurate-phylogeny-reconstruction-using" target="_blank" rel="external">Fast and accurate phylogeny reconstruction using filtered spaced-word matches</a></h4><p>Bioinformatics (2017) 33 (7): 971-979.<br>不依赖于比对的系统发育树构建。</p>
<h4 id="Improved-VCF-normalization-for-accurate-VCF-comparison"><a href="#Improved-VCF-normalization-for-accurate-VCF-comparison" class="headerlink" title="Improved VCF normalization for accurate VCF comparison"></a><a href="https://academic.oup.com/bioinformatics/article-abstract/33/7/964/2623048/Improved-VCF-normalization-for-accurate-VCF?redirectedFrom=fulltext" target="_blank" rel="external">Improved VCF normalization for accurate VCF comparison</a></h4><p>Bioinformatics (2017) 33 (7): 964-970.<br>该工具将VCF文件中所有的变异替换到参考序列中，生成单个样本的基因组序列。再重新call variants，生成标准的格式，便于比较。［这个比较有意思，对于indel，即使同样的变异，samtools和gatk纪录的形式不同，没法比较，通过该工具可能可以解决。］</p>
<h4 id="Biomartr-genomic-data-retrieval-with-R"><a href="#Biomartr-genomic-data-retrieval-with-R" class="headerlink" title="Biomartr: genomic data retrieval with R"></a><a href="https://academic.oup.com/bioinformatics/article/33/8/1216/2931816/Biomartr-genomic-data-retrieval-with-R" target="_blank" rel="external">Biomartr: genomic data retrieval with R</a></h4><p>Bioinformatics (2017) 33 (8): 1216-1217.<br>用来从NCBI上面下载数据（参考序列，数据库等）的R包。</p>
<h4 id="GWAlpha-genome-wide-estimation-of-additive-effects-alpha-based-on-trait-quantile-distribution-from-pool-sequencing-experiments"><a href="#GWAlpha-genome-wide-estimation-of-additive-effects-alpha-based-on-trait-quantile-distribution-from-pool-sequencing-experiments" class="headerlink" title="GWAlpha: genome-wide estimation of additive effects (alpha) based on trait quantile distribution from pool-sequencing experiments"></a><a href="https://academic.oup.com/bioinformatics/article-abstract/33/8/1246/2729762/GWAlpha-genome-wide-estimation-of-additive-effects?redirectedFrom=fulltext" target="_blank" rel="external">GWAlpha: genome-wide estimation of additive effects (alpha) based on trait quantile distribution from pool-sequencing experiments</a></h4><p>Bioinformatics (2017) 33 (8): 1246-1247.<br>通过混池测序，估计加性效应值。</p>
<h4 id="Human-Demographic-History-Impacts-Genetic-Risk-Prediction-across-Diverse-Populations30107-6"><a href="#Human-Demographic-History-Impacts-Genetic-Risk-Prediction-across-Diverse-Populations30107-6" class="headerlink" title="Human Demographic History Impacts Genetic Risk Prediction across Diverse Populations30107-6)"></a><a href="http://www.cell.com/ajhg/abstract/S0002-9297\(17\" target="_blank" rel="external">Human Demographic History Impacts Genetic Risk Prediction across Diverse Populations</a>30107-6)</h4><p>AJHG 100, 635–649, April 6, 2017<br>人类群体历史影响遗传风险预测。在一个群体中预测的疾病相关位点，也许并不适用于另一个群体。</p>
<h4 id="De-novo-assembly-of-the-Aedes-aegypti-genome-using-Hi-C-yields-chromosome-length-scaffolds"><a href="#De-novo-assembly-of-the-Aedes-aegypti-genome-using-Hi-C-yields-chromosome-length-scaffolds" class="headerlink" title="De novo assembly of the Aedes aegypti genome using Hi-C yields chromosome-length scaffolds"></a><a href="http://science.sciencemag.org/content/356/6333/92" target="_blank" rel="external">De novo assembly of the Aedes aegypti genome using Hi-C yields chromosome-length scaffolds</a></h4><p>Science  07 Apr 2017: Vol. 356, Issue 6333, pp. 92-95<br>利用Hi-C技术辅助组装，将伊蚊基因组组装到染色体水平。</p>
<h4 id="A-chromosome-conformation-capture-ordered-sequence-of-the-barley-genome"><a href="#A-chromosome-conformation-capture-ordered-sequence-of-the-barley-genome" class="headerlink" title="A chromosome conformation capture ordered sequence of the barley genome"></a><a href="http://www.nature.com/nature/journal/v544/n7651/full/nature22043.html#author-information" target="_blank" rel="external">A chromosome conformation capture ordered sequence of the barley genome</a></h4><p>Nature 544, 427–433 (27 April 2017)<br>大麦基因组发表</p>
<h4 id="SynthEx-a-synthetic-normal-based-DNA-sequencing-tool-for-copy-number-alteration-detection-and-tumor-heterogeneity-profiling"><a href="#SynthEx-a-synthetic-normal-based-DNA-sequencing-tool-for-copy-number-alteration-detection-and-tumor-heterogeneity-profiling" class="headerlink" title="SynthEx: a synthetic-normal-based DNA sequencing tool for copy number alteration detection and tumor heterogeneity profiling"></a><a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1193-3" target="_blank" rel="external">SynthEx: a synthetic-normal-based DNA sequencing tool for copy number alteration detection and tumor heterogeneity profiling</a></h4><p>Genome Biology 2017<br>基于全基因组或全外显子组测序的拷贝数变异检测工具。其能力和基于芯片的软件相当，胜过其它基于测序的软件。</p>
<h4 id="Ancient-genomic-changes-associated-with-domestication-of-the-horse"><a href="#Ancient-genomic-changes-associated-with-domestication-of-the-horse" class="headerlink" title="Ancient genomic changes associated with domestication of the horse"></a><a href="http://science.sciencemag.org/content/356/6336/442" target="_blank" rel="external">Ancient genomic changes associated with domestication of the horse</a></h4><p>Science  28 Apr 2017<br>研究人员对14匹距今2000～4000年的古马进行全基因组测序，揭示马的驯化过程。</p>
<h4 id="Genetic-signatures-of-high-altitude-adaptation-in-Tibetans"><a href="#Genetic-signatures-of-high-altitude-adaptation-in-Tibetans" class="headerlink" title="Genetic signatures of high-altitude adaptation in Tibetans"></a><a href="http://www.pnas.org/content/114/16/4189.full" target="_blank" rel="external">Genetic signatures of high-altitude adaptation in Tibetans</a></h4><p>PNAS April 18, 2017 vol. 114 no. 16 4189-4194<br>文章结合了3,008个藏族人7.3 M SNP 芯片数据以及7,287个非藏族东亚人数据，揭示了藏族人适应高原环境基因。</p>
<h4 id="Parallel-adaptive-evolution-of-geographically-distant-herring-populations-on-both-sides-of-the-North-Atlantic-Ocean"><a href="#Parallel-adaptive-evolution-of-geographically-distant-herring-populations-on-both-sides-of-the-North-Atlantic-Ocean" class="headerlink" title="Parallel adaptive evolution of geographically distant herring populations on both sides of the North Atlantic Ocean"></a><a href="http://www.pnas.org/content/114/17/E3452.full" target="_blank" rel="external">Parallel adaptive evolution of geographically distant herring populations on both sides of the North Atlantic Ocean</a></h4><p>PNAS April 25, 2017 vol. 114 no. 17 E3452-E3461<br>北大西洋鲱鱼的平行进化。</p>
<h4 id="Finding-a-Needle-in-a-Haystack-Distinguishing-Mexican-Maize-Landraces-Using-a-Small-Number-of-SNPs"><a href="#Finding-a-Needle-in-a-Haystack-Distinguishing-Mexican-Maize-Landraces-Using-a-Small-Number-of-SNPs" class="headerlink" title="Finding a Needle in a Haystack: Distinguishing Mexican Maize Landraces Using a Small Number of SNPs"></a><a href="http://journal.frontiersin.org/article/10.3389/fgene.2017.00045/full" target="_blank" rel="external">Finding a Needle in a Haystack: Distinguishing Mexican Maize Landraces Using a Small Number of SNPs</a></h4><p>Front. Genet., 18 April 2017<br>作者通过50个样本（5个不同地方种）的50k芯片数据，筛选出了用于区分不同地方种的SNPs。“we identified 20 landrace-informative SNPs and 14 altitude-informative SNPs.”</p>
<h4 id="Full-Chloroplast-Genome-Assembly-of-11-Diverse-Watermelon-Accessions"><a href="#Full-Chloroplast-Genome-Assembly-of-11-Diverse-Watermelon-Accessions" class="headerlink" title="Full Chloroplast Genome Assembly of 11 Diverse Watermelon Accessions"></a><a href="http://journal.frontiersin.org/article/10.3389/fgene.2017.00046/full" target="_blank" rel="external">Full Chloroplast Genome Assembly of 11 Diverse Watermelon Accessions</a></h4><p>Front. Genet., 18 April 2017<br>11个西瓜全叶绿体基因组序列。</p>
<h4 id="Analytical-Biases-Associated-with-GC-Content-in-Molecular-Evolution"><a href="#Analytical-Biases-Associated-with-GC-Content-in-Molecular-Evolution" class="headerlink" title="Analytical Biases Associated with GC-Content in Molecular Evolution"></a><a href="http://journal.frontiersin.org/article/10.3389/fgene.2017.00016/full" target="_blank" rel="external">Analytical Biases Associated with GC-Content in Molecular Evolution</a></h4><p>Front. Genet., 15 February 2017<br>碱基组成(GC含量)在不同位点或分类群的异质性，会使分子进化分析产生偏好性，如构建系统发育树，检测自然选择或估计密码子使用率。文章对产生偏好的原因以及避开偏好的方法进行了综述。</p>
<h4 id="A-genome-wide-association-study-identifies-six-novel-risk-loci-for-primary-biliary-cholangitis"><a href="#A-genome-wide-association-study-identifies-six-novel-risk-loci-for-primary-biliary-cholangitis" class="headerlink" title="A genome-wide association study identifies six novel risk loci for primary biliary cholangitis"></a><a href="https://www.nature.com/articles/ncomms14828" target="_blank" rel="external">A genome-wide association study identifies six novel risk loci for primary biliary cholangitis</a></h4><p>Nature Communications 8, Article number: 14828 (2017)<br>全基因组关联分析鉴定6个新的原发胆汁性胆管炎风险位点。</p>
<h4 id="Genome-assembly-with-in-vitro-proximity-ligation-data-and-whole-genome-triplication-in-lettuce"><a href="#Genome-assembly-with-in-vitro-proximity-ligation-data-and-whole-genome-triplication-in-lettuce" class="headerlink" title="Genome assembly with in vitro proximity ligation data and whole-genome triplication in lettuce"></a><a href="http://www.nature.com/articles/ncomms14953" target="_blank" rel="external">Genome assembly with in vitro proximity ligation data and whole-genome triplication in lettuce</a></h4><p>Nature Communications 8, Article number: 14953 (2017)<br>生菜基因组，2.7 G。用in vitro proximity ligation数据锚定小scaffold。</p>
<h4 id="Reconstructing-the-genome-of-the-most-recent-common-ancestor-of-flowering-plants"><a href="#Reconstructing-the-genome-of-the-most-recent-common-ancestor-of-flowering-plants" class="headerlink" title="Reconstructing the genome of the most recent common ancestor of flowering plants"></a><a href="http://www.nature.com/ng/journal/v49/n4/abs/ng.3813.html" target="_blank" rel="external">Reconstructing the genome of the most recent common ancestor of flowering plants</a></h4><p>Nature Genetics 49, 490–496 (2017)<br>开花植物最近共同祖先基因组构建。</p>
<h4 id="Asymmetric-subgenome-selection-and-cis-regulatory-divergence-during-cotton-domestication"><a href="#Asymmetric-subgenome-selection-and-cis-regulatory-divergence-during-cotton-domestication" class="headerlink" title="Asymmetric subgenome selection and cis-regulatory divergence during cotton domestication"></a><a href="http://www.nature.com/ng/journal/v49/n4/abs/ng.3807.html" target="_blank" rel="external">Asymmetric subgenome selection and cis-regulatory divergence during cotton domestication</a></h4><p>Nature Genetics 49, 579–587 (2017)<br>野生和栽培棉花揭示驯化过程中受选择基因。</p>
<h4 id="Fast-scalable-prediction-of-deleterious-noncoding-variants-from-functional-and-population-genomic-data"><a href="#Fast-scalable-prediction-of-deleterious-noncoding-variants-from-functional-and-population-genomic-data" class="headerlink" title="Fast, scalable prediction of deleterious noncoding variants from functional and population genomic data"></a><a href="http://www.nature.com/ng/journal/v49/n4/abs/ng.3810.html" target="_blank" rel="external">Fast, scalable prediction of deleterious noncoding variants from functional and population genomic data</a></h4><p>Nature Genetics 49, 618–624 (2017)<br>有害非编码突变预测。（是否适合非模式生物？）</p>
<h4 id="Single-molecule-sequencing-and-chromatin-conformation-capture-enable-de-novo-reference-assembly-of-the-domestic-goat-genome"><a href="#Single-molecule-sequencing-and-chromatin-conformation-capture-enable-de-novo-reference-assembly-of-the-domestic-goat-genome" class="headerlink" title="Single-molecule sequencing and chromatin conformation capture enable de novo reference assembly of the domestic goat genome"></a><a href="http://www.nature.com/ng/journal/v49/n4/abs/ng.3802.html" target="_blank" rel="external">Single-molecule sequencing and chromatin conformation capture enable de novo reference assembly of the domestic goat genome</a></h4><p>Nature Genetics 49, 643–650 (2017)<br>通过单分子测序和染色体构像捕获组装山羊基因组。</p>
<h4 id="Scallop-genome-provides-insights-into-evolution-of-bilaterian-karyotype-and-development"><a href="#Scallop-genome-provides-insights-into-evolution-of-bilaterian-karyotype-and-development" class="headerlink" title="Scallop genome provides insights into evolution of bilaterian karyotype and development"></a><a href="http://www.nature.com/articles/s41559-017-0120" target="_blank" rel="external">Scallop genome provides insights into evolution of bilaterian karyotype and development</a></h4><p>Nature Ecology &amp; Evolution 1, Article number: 0120 (2017)<br>扇贝基因组。</p>

      
    </div>

    <div>
      
        

      
    </div>

    <div>
      
        
  <div style="padding: 10px 0; margin: 20px auto; width: 90%; text-align: center;">
    <div>如果我的博客能帮你节约时间，我会非常高兴你请我喝杯咖啡。</div>
    <button id="rewardButton" disable="enable" onclick="var qr = document.getElementById('QR'); if (qr.style.display === 'none') {qr.style.display='block';} else {qr.style.display='none'}">
      <span>赏</span>
    </button>
    <div id="QR" style="display: none;">
      
        <div id="wechat" style="display: inline-block">
          <img id="wechat_qr" src="http://onzjn6hm6.bkt.clouddn.com/weixin.jpg" alt="Coly Fu WeChat Pay"/>
          <p>微信打赏</p>
        </div>
      
      
        <div id="alipay" style="display: inline-block">
          <img id="alipay_qr" src="http://onzjn6hm6.bkt.clouddn.com/alipay.jpg" alt="Coly Fu Alipay"/>
          <p>支付宝打赏</p>
        </div>
      
    </div>
  </div>


      
    </div>

    <div>
      
        

      
    </div>

    <footer class="post-footer">
      
        <div class="post-tags">
          
            <a href="/tags/文献导读/" rel="tag"># 文献导读</a>
          
        </div>
      

      
        
      

      
        <div class="post-nav">
          <div class="post-nav-next post-nav-item">
            
              <a href="/Population-Genetics/haplotype-network.html" rel="next" title="单体型网络（Haplotype Network）构建及展示">
                <i class="fa fa-chevron-left"></i> 单体型网络（Haplotype Network）构建及展示
              </a>
            
          </div>

          <span class="post-nav-divider"></span>

          <div class="post-nav-prev post-nav-item">
            
              <a href="/QC/fastq-phred-trans.html" rel="prev" title="判断fastq文件质量编码格式及Phred64转Phred33方法">
                判断fastq文件质量编码格式及Phred64转Phred33方法 <i class="fa fa-chevron-right"></i>
              </a>
            
          </div>
        </div>
      

      
      
    </footer>
  </article>



    <div class="post-spread">
      
    </div>
  </div>


          </div>
          


          
  <div class="comments" id="comments">
    
      <div id="lv-container" data-id="city" data-uid="MTAyMC8yODA3OC80NjUx"></div>
    
  </div>


        </div>
        
          
  
  <div class="sidebar-toggle">
    <div class="sidebar-toggle-line-wrap">
      <span class="sidebar-toggle-line sidebar-toggle-line-first"></span>
      <span class="sidebar-toggle-line sidebar-toggle-line-middle"></span>
      <span class="sidebar-toggle-line sidebar-toggle-line-last"></span>
    </div>
  </div>

  <aside id="sidebar" class="sidebar">
    <div class="sidebar-inner">

      

      
        <ul class="sidebar-nav motion-element">
          <li class="sidebar-nav-toc sidebar-nav-active" data-target="post-toc-wrap" >
            文章目录
          </li>
          <li class="sidebar-nav-overview" data-target="site-overview">
            站点概览
          </li>
        </ul>
      

      <section class="site-overview sidebar-panel">
        <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
          <img class="site-author-image" itemprop="image"
               src="http://onzjn6hm6.bkt.clouddn.com/avatar.jpg"
               alt="Coly Fu" />
          <p class="site-author-name" itemprop="name">Coly Fu</p>
           
              <p class="site-description motion-element" itemprop="description"></p>
           
        </div>
        <nav class="site-state motion-element">

          
            <div class="site-state-item site-state-posts">
              <a href="/archives">
                <span class="site-state-item-count">18</span>
                <span class="site-state-item-name">日志</span>
              </a>
            </div>
          

          
            
            
            <div class="site-state-item site-state-categories">
              <a href="/categories/index.html">
                <span class="site-state-item-count">6</span>
                <span class="site-state-item-name">分类</span>
              </a>
            </div>
          

          
            
            
            <div class="site-state-item site-state-tags">
              <a href="/tags/index.html">
                <span class="site-state-item-count">33</span>
                <span class="site-state-item-name">标签</span>
              </a>
            </div>
          

        </nav>

        
          <div class="feed-link motion-element">
            <a href="/atom.xml" rel="alternate">
              <i class="fa fa-rss"></i>
              RSS
            </a>
          </div>
        

        <div class="links-of-author motion-element">
          
        </div>

        
        

        
        

        


      </section>

      
      <!--noindex-->
        <section class="post-toc-wrap motion-element sidebar-panel sidebar-panel-active">
          <div class="post-toc">

            
              
            

            
              <div class="post-toc-content"><ol class="nav"><li class="nav-item nav-level-4"><a class="nav-link" href="#The-Rice-Paradox-Multiple-Origins-but-Single-Domestication-in-Asian-Rice"><span class="nav-number">1.</span> <span class="nav-text">The Rice Paradox: Multiple Origins but Single Domestication in Asian Rice</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Genome-Sequencing-Reveals-the-Origin-of-the-Allotetraploid-Arabidopsis-suecica"><span class="nav-number">2.</span> <span class="nav-text">Genome Sequencing Reveals the Origin of the Allotetraploid Arabidopsis suecica</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Deleterious-Variants-in-Asian-Rice-and-the-Potential-Cost-of-Domestication"><span class="nav-number">3.</span> <span class="nav-text">Deleterious Variants in Asian Rice and the Potential Cost of Domestication</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#A-Working-Model-of-the-Deep-Relationships-of-Diverse-Modern-Human-Genetic-Lineages-Outside-of-Africa"><span class="nav-number">4.</span> <span class="nav-text">A Working Model of the Deep Relationships of Diverse Modern Human Genetic Lineages Outside of Africa </span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#What-is-adaptation-by-natural-selection-Perspectives-of-an-experimental-microbiologist"><span class="nav-number">5.</span> <span class="nav-text">What is adaptation by natural selection? Perspectives of an experimental microbiologist</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Evolutionary-history-of-Tibetans-inferred-from-whole-genome-sequencing"><span class="nav-number">6.</span> <span class="nav-text">Evolutionary history of Tibetans inferred from whole-genome sequencing</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#The-time-and-place-of-European-admixture-in-Ashkenazi-Jewish-history"><span class="nav-number">7.</span> <span class="nav-text">The time and place of European admixture in Ashkenazi Jewish history</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#RECKONER-read-error-corrector-based-on-KMC"><span class="nav-number">8.</span> <span class="nav-text">RECKONER: read error corrector based on KMC</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#SVScore-an-impact-prediction-tool-for-structural-variation"><span class="nav-number">9.</span> <span class="nav-text">SVScore: an impact prediction tool for structural variation</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Fast-and-accurate-phylogeny-reconstruction-using-filtered-spaced-word-matches"><span class="nav-number">10.</span> <span class="nav-text">Fast and accurate phylogeny reconstruction using filtered spaced-word matches</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Improved-VCF-normalization-for-accurate-VCF-comparison"><span class="nav-number">11.</span> <span class="nav-text">Improved VCF normalization for accurate VCF comparison</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Biomartr-genomic-data-retrieval-with-R"><span class="nav-number">12.</span> <span class="nav-text">Biomartr: genomic data retrieval with R</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#GWAlpha-genome-wide-estimation-of-additive-effects-alpha-based-on-trait-quantile-distribution-from-pool-sequencing-experiments"><span class="nav-number">13.</span> <span class="nav-text">GWAlpha: genome-wide estimation of additive effects (alpha) based on trait quantile distribution from pool-sequencing experiments</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Human-Demographic-History-Impacts-Genetic-Risk-Prediction-across-Diverse-Populations30107-6"><span class="nav-number">14.</span> <span class="nav-text">Human Demographic History Impacts Genetic Risk Prediction across Diverse Populations30107-6)</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#De-novo-assembly-of-the-Aedes-aegypti-genome-using-Hi-C-yields-chromosome-length-scaffolds"><span class="nav-number">15.</span> <span class="nav-text">De novo assembly of the Aedes aegypti genome using Hi-C yields chromosome-length scaffolds</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#A-chromosome-conformation-capture-ordered-sequence-of-the-barley-genome"><span class="nav-number">16.</span> <span class="nav-text">A chromosome conformation capture ordered sequence of the barley genome</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#SynthEx-a-synthetic-normal-based-DNA-sequencing-tool-for-copy-number-alteration-detection-and-tumor-heterogeneity-profiling"><span class="nav-number">17.</span> <span class="nav-text">SynthEx: a synthetic-normal-based DNA sequencing tool for copy number alteration detection and tumor heterogeneity profiling</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Ancient-genomic-changes-associated-with-domestication-of-the-horse"><span class="nav-number">18.</span> <span class="nav-text">Ancient genomic changes associated with domestication of the horse</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Genetic-signatures-of-high-altitude-adaptation-in-Tibetans"><span class="nav-number">19.</span> <span class="nav-text">Genetic signatures of high-altitude adaptation in Tibetans</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Parallel-adaptive-evolution-of-geographically-distant-herring-populations-on-both-sides-of-the-North-Atlantic-Ocean"><span class="nav-number">20.</span> <span class="nav-text">Parallel adaptive evolution of geographically distant herring populations on both sides of the North Atlantic Ocean</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Finding-a-Needle-in-a-Haystack-Distinguishing-Mexican-Maize-Landraces-Using-a-Small-Number-of-SNPs"><span class="nav-number">21.</span> <span class="nav-text">Finding a Needle in a Haystack: Distinguishing Mexican Maize Landraces Using a Small Number of SNPs</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Full-Chloroplast-Genome-Assembly-of-11-Diverse-Watermelon-Accessions"><span class="nav-number">22.</span> <span class="nav-text">Full Chloroplast Genome Assembly of 11 Diverse Watermelon Accessions</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Analytical-Biases-Associated-with-GC-Content-in-Molecular-Evolution"><span class="nav-number">23.</span> <span class="nav-text">Analytical Biases Associated with GC-Content in Molecular Evolution</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#A-genome-wide-association-study-identifies-six-novel-risk-loci-for-primary-biliary-cholangitis"><span class="nav-number">24.</span> <span class="nav-text">A genome-wide association study identifies six novel risk loci for primary biliary cholangitis</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Genome-assembly-with-in-vitro-proximity-ligation-data-and-whole-genome-triplication-in-lettuce"><span class="nav-number">25.</span> <span class="nav-text">Genome assembly with in vitro proximity ligation data and whole-genome triplication in lettuce</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Reconstructing-the-genome-of-the-most-recent-common-ancestor-of-flowering-plants"><span class="nav-number">26.</span> <span class="nav-text">Reconstructing the genome of the most recent common ancestor of flowering plants</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Asymmetric-subgenome-selection-and-cis-regulatory-divergence-during-cotton-domestication"><span class="nav-number">27.</span> <span class="nav-text">Asymmetric subgenome selection and cis-regulatory divergence during cotton domestication</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Fast-scalable-prediction-of-deleterious-noncoding-variants-from-functional-and-population-genomic-data"><span class="nav-number">28.</span> <span class="nav-text">Fast, scalable prediction of deleterious noncoding variants from functional and population genomic data</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Single-molecule-sequencing-and-chromatin-conformation-capture-enable-de-novo-reference-assembly-of-the-domestic-goat-genome"><span class="nav-number">29.</span> <span class="nav-text">Single-molecule sequencing and chromatin conformation capture enable de novo reference assembly of the domestic goat genome</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#Scallop-genome-provides-insights-into-evolution-of-bilaterian-karyotype-and-development"><span class="nav-number">30.</span> <span class="nav-text">Scallop genome provides insights into evolution of bilaterian karyotype and development</span></a></li></ol></div>
            

          </div>
        </section>
      <!--/noindex-->
      

      

    </div>
  </aside>


        
      </div>
    </main>

    <footer id="footer" class="footer">
      <div class="footer-inner">
        <div class="copyright" >
  
  &copy;  2016 - 
  <span itemprop="copyrightYear">2018</span>
  <span class="with-love">
    <i class="fa fa-heart"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">Coly Fu</span>
</div>



        

        
      </div>
    </footer>

    
      <div class="back-to-top">
        <i class="fa fa-arrow-up"></i>
        
      </div>
    

  </div>

  

<script type="text/javascript">
  if (Object.prototype.toString.call(window.Promise) !== '[object Function]') {
    window.Promise = null;
  }
</script>









  






  
  <script type="text/javascript" src="/lib/jquery/index.js?v=2.1.3"></script>

  
  <script type="text/javascript" src="/lib/fastclick/lib/fastclick.min.js?v=1.0.6"></script>

  
  <script type="text/javascript" src="/lib/jquery_lazyload/jquery.lazyload.js?v=1.9.7"></script>

  
  <script type="text/javascript" src="/lib/velocity/velocity.min.js?v=1.2.1"></script>

  
  <script type="text/javascript" src="/lib/velocity/velocity.ui.min.js?v=1.2.1"></script>

  
  <script type="text/javascript" src="/lib/fancybox/source/jquery.fancybox.pack.js?v=2.1.5"></script>


  


  <script type="text/javascript" src="/js/src/utils.js?v=5.1.0"></script>

  <script type="text/javascript" src="/js/src/motion.js?v=5.1.0"></script>



  
  

  
  <script type="text/javascript" src="/js/src/scrollspy.js?v=5.1.0"></script>
<script type="text/javascript" src="/js/src/post-details.js?v=5.1.0"></script>



  


  <script type="text/javascript" src="/js/src/bootstrap.js?v=5.1.0"></script>



  


  




	





  





  





  
    <script type="text/javascript">
      (function(d, s) {
        var j, e = d.getElementsByTagName(s)[0];
        if (typeof LivereTower === 'function') { return; }
        j = d.createElement(s);
        j.src = 'https://cdn-city.livere.com/js/embed.dist.js';
        j.async = true;
        e.parentNode.insertBefore(j, e);
      })(document, 'script');
    </script>
  






  

  <script type="text/javascript">
    // Popup Window;
    var isfetched = false;
    // Search DB path;
    var search_path = "search.xml";
    if (search_path.length == 0) {
      search_path = "search.xml";
    }
    var path = "/" + search_path;
    // monitor main search box;

    function proceedsearch() {
      $("body")
        .append('<div class="search-popup-overlay local-search-pop-overlay"></div>')
        .css('overflow', 'hidden');
      $('.popup').toggle();
    }
    // search function;
    var searchFunc = function(path, search_id, content_id) {
      'use strict';
      $.ajax({
        url: path,
        dataType: "xml",
        async: true,
        success: function( xmlResponse ) {
          // get the contents from search data
          isfetched = true;
          $('.popup').detach().appendTo('.header-inner');
          var datas = $( "entry", xmlResponse ).map(function() {
            return {
              title: $( "title", this ).text(),
              content: $("content",this).text(),
              url: $( "url" , this).text()
            };
          }).get();
          var $input = document.getElementById(search_id);
          var $resultContent = document.getElementById(content_id);
          $input.addEventListener('input', function(){
            var matchcounts = 0;
            var str='<ul class=\"search-result-list\">';
            var keywords = this.value.trim().toLowerCase().split(/[\s\-]+/);
            $resultContent.innerHTML = "";
            if (this.value.trim().length > 1) {
              // perform local searching
              datas.forEach(function(data) {
                var isMatch = false;
                var content_index = [];
                var data_title = data.title.trim().toLowerCase();
                var data_content = data.content.trim().replace(/<[^>]+>/g,"").toLowerCase();
                var data_url = decodeURIComponent(data.url);
                var index_title = -1;
                var index_content = -1;
                var first_occur = -1;
                // only match artiles with not empty titles and contents
                if(data_title != '') {
                  keywords.forEach(function(keyword, i) {
                    index_title = data_title.indexOf(keyword);
                    index_content = data_content.indexOf(keyword);
                    if( index_title >= 0 || index_content >= 0 ){
                      isMatch = true;
                      if (i == 0) {
                        first_occur = index_content;
                      }
                    }

                  });
                }
                // show search results
                if (isMatch) {
                  matchcounts += 1;
                  str += "<li><a href='"+ data_url +"' class='search-result-title'>"+ data_title +"</a>";
                  var content = data.content.trim().replace(/<[^>]+>/g,"");
                  if (first_occur >= 0) {
                    // cut out 100 characters
                    var start = first_occur - 20;
                    var end = first_occur + 80;
                    if(start < 0){
                      start = 0;
                    }
                    if(start == 0){
                      end = 50;
                    }
                    if(end > content.length){
                      end = content.length;
                    }
                    var match_content = content.substring(start, end);
                    // highlight all keywords
                    keywords.forEach(function(keyword){
                      var regS = new RegExp(keyword, "gi");
                      match_content = match_content.replace(regS, "<b class=\"search-keyword\">"+keyword+"</b>");
                    });

                    str += "<p class=\"search-result\">" + match_content +"...</p>"
                  }
                  str += "</li>";
                }
              })};
            str += "</ul>";
            if (matchcounts == 0) { str = '<div id="no-result"><i class="fa fa-frown-o fa-5x" /></div>' }
            if (keywords == "") { str = '<div id="no-result"><i class="fa fa-search fa-5x" /></div>' }
            $resultContent.innerHTML = str;
          });
          proceedsearch();
        }
      });}

    // handle and trigger popup window;
    $('.popup-trigger').click(function(e) {
      e.stopPropagation();
      if (isfetched == false) {
        searchFunc(path, 'local-search-input', 'local-search-result');
      } else {
        proceedsearch();
      };
    });

    $('.popup-btn-close').click(function(e){
      $('.popup').hide();
      $(".local-search-pop-overlay").remove();
      $('body').css('overflow', '');
    });
    $('.popup').click(function(e){
      e.stopPropagation();
    });
  </script>





  

  
  <script src="https://cdn1.lncld.net/static/js/av-core-mini-0.6.1.js"></script>
  <script>AV.initialize("TYEr3gWwGEom2g4BHIufivSY-gzGzoHsz", "76KRaa6vXccIYhlGecUXpv6W");</script>
  <script>
    function showTime(Counter) {
      var query = new AV.Query(Counter);
      var entries = [];
      var $visitors = $(".leancloud_visitors");

      $visitors.each(function () {
        entries.push( $(this).attr("id").trim() );
      });

      query.containedIn('url', entries);
      query.find()
        .done(function (results) {
          var COUNT_CONTAINER_REF = '.leancloud-visitors-count';

          if (results.length === 0) {
            $visitors.find(COUNT_CONTAINER_REF).text(0);
            return;
          }

          for (var i = 0; i < results.length; i++) {
            var item = results[i];
            var url = item.get('url');
            var time = item.get('time');
            var element = document.getElementById(url);

            $(element).find(COUNT_CONTAINER_REF).text(time);
          }
          for(var i = 0; i < entries.length; i++) {
            var url = entries[i];
            var element = document.getElementById(url);
            var countSpan = $(element).find(COUNT_CONTAINER_REF);
            if( countSpan.text() == '') {
              countSpan.text(0);
            }
          }
        })
        .fail(function (object, error) {
          console.log("Error: " + error.code + " " + error.message);
        });
    }

    function addCount(Counter) {
      var $visitors = $(".leancloud_visitors");
      var url = $visitors.attr('id').trim();
      var title = $visitors.attr('data-flag-title').trim();
      var query = new AV.Query(Counter);

      query.equalTo("url", url);
      query.find({
        success: function(results) {
          if (results.length > 0) {
            var counter = results[0];
            counter.fetchWhenSave(true);
            counter.increment("time");
            counter.save(null, {
              success: function(counter) {
                var $element = $(document.getElementById(url));
                $element.find('.leancloud-visitors-count').text(counter.get('time'));
              },
              error: function(counter, error) {
                console.log('Failed to save Visitor num, with error message: ' + error.message);
              }
            });
          } else {
            var newcounter = new Counter();
            /* Set ACL */
            var acl = new AV.ACL();
            acl.setPublicReadAccess(true);
            acl.setPublicWriteAccess(true);
            newcounter.setACL(acl);
            /* End Set ACL */
            newcounter.set("title", title);
            newcounter.set("url", url);
            newcounter.set("time", 1);
            newcounter.save(null, {
              success: function(newcounter) {
                var $element = $(document.getElementById(url));
                $element.find('.leancloud-visitors-count').text(newcounter.get('time'));
              },
              error: function(newcounter, error) {
                console.log('Failed to create');
              }
            });
          }
        },
        error: function(error) {
          console.log('Error:' + error.code + " " + error.message);
        }
      });
    }

    $(function() {
      var Counter = AV.Object.extend("Counter");
      if ($('.leancloud_visitors').length == 1) {
        addCount(Counter);
      } else if ($('.post-title-link').length > 1) {
        showTime(Counter);
      }
    });
  </script>



  
<script>
(function(){
    var bp = document.createElement('script');
    var curProtocol = window.location.protocol.split(':')[0];
    if (curProtocol === 'https') {
        bp.src = 'https://zz.bdstatic.com/linksubmit/push.js';        
    }
    else {
        bp.src = 'http://push.zhanzhang.baidu.com/push.js';
    }
    var s = document.getElementsByTagName("script")[0];
    s.parentNode.insertBefore(bp, s);
})();
</script>


  

  

</body>
</html>
